packages/video_player/closed_caption_file/lib/closed_caption_file.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library closed_caption_file;
+
+import 'src/sub_rip.dart';
+
+/// A structured representation of a parsed closed caption file.
+///
+/// A closed caption file includes a list of captions, each with a start and end
+/// time for when the given closed caption should be displayed.
+///
+/// The [captions] are a list of all captions in a file, in the order that they
+/// appeared in the file.
+class ClosedCaptionFile {
+  /// The full list of captions from a given file.
```

Comment:
nit: Probably worth saying here whether or not the list is expected to be ordered from start to end duration.

Reply:
I don't know enough about the different formats to say for sure that they will always be sorted by duration. 

For now, I'm putting that they will be in the order that they appear in the file.

Reply:
Sounds good, as long as it's clear what's expected.